iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/helper/DummyAndroidRunner.java

```diff
@@ -5,28 +5,13 @@
 
 package com.microsoft.azure.sdk.iot.android.helper;
 
-import com.microsoft.appcenter.espresso.Factory;
-import com.microsoft.appcenter.espresso.ReportHelper;
-import com.microsoft.azure.sdk.iot.common.helpers.BasicTierOnlyRule;
-import com.microsoft.azure.sdk.iot.common.helpers.ConditionalIgnoreRule;
-
-import org.junit.After;
-import org.junit.Rule;
 import org.junit.Test;
 
-// This class is to add a dummy test for every TestGroup.  Currently, App Center errors out
-// when a TestGroup did not execute any actual tests.
+// This class is to add a dummy test for every TestGroup.  Currently, android testing errors out
+// when a TestGroup did not execute any actual tests, so this forces each test group to have at least
+// this test
 public class DummyAndroidRunner
```

Comment:
Is this really a "dummy"? Else we might want to rename it to something else

Reply:
Sure, I can rename this to "Fake"

Reply:
It exists only so that, if all other tests get filtered, at least one test runs. Android instrumentation testing fails if no tests run, counterintuitively